xla/stream_executor/cuda/cuda_driver.cc

```diff
@@ -776,7 +776,7 @@ GpuDriver::GraphAddNode(CUgraphNode* node, CUgraph graph,
 }
 
 /* static */ tsl::Status GpuDriver::GraphAddKernelNode(
-    CUgraphNode* node, CUgraph graph, absl::Span<CUgraphNode> deps,
+    CUgraphNode* node, CUgraph graph, absl::Span<GpuGraphNodeHandle> deps,
```

Comment:
should be `CUgraphNode` for insise `cuda_driver` (in general all `GpuXYZ` should be replaced by CUDA types)